scripts/add_gen3_repo.py

```diff
@@ -117,6 +117,14 @@ def _export_for_copy(dataset, repo):
         # Need all detectors, even those without data, for visit definition
         contents.saveDataIds(butler.registry.queryDataIds({"detector"}).expanded())
         contents.saveDatasets(butler.registry.queryDatasets(datasetType=..., collections=...))
+        # Explicitly save the calibration collection.
+        # We filter on collection type rather than name to insulate a bit from
+        # naming convention changes.  We need to _not_ include the RUN
+        # collections here because that will export an empty collection that
+        # looks like it should contain raws, and that's enough to confuse
+        # ap_verify logic that guards against ingesting raws twice.
```

Comment:
Could this be made a bit more concise?
```suggestion
        # Explicitly save the calibration collection.
        # Do _not_ include the RUN collections here because that will export
        # an empty raws collection, which ap_verify assumes does not exist
        # before ingest.
```